### DISCUSSION TITLE
Installation Error : MODULE_NOT_FOUND @directus/shared/utils

### DISCUSSION THREAD

**Discussion by jnachman123** [directus/directus#7485] [Archive]
![image](https://user-images.githubusercontent.com/44912271/129885271-741b7545-f79b-4062-8eb3-f8e51efec22e.png)

Get this after running npm create-directus-project xxxxxxx

any ideas, stuck at the moment. any help will be appreciated, running this on my windows machine via terminal on vs code. 

Thanks in advance.

**Comment by nickrum** [comment]
Directus requires at least Node v12.20 to run.
